wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -5,31 +5,51 @@
 
 		<div class="article__content">
 			<article class="quiz-creator-product-page" id="tool-landing-page">
-				<section class="hero aqua" style="margin-top: -36px;">
-					<div class="container container--xl wrapper--pad">
-							<div class="quiz-creator-intro-copy">
-								<h1>Create interactive, embeddable quizzes that are proven to increase engagement online.</h1>
-								<a href="/research/online-polls-and-quizzes/">Read the report →</a>
+				<section class="hero aqua" style="margin-top: -36px; margin-bottom:0px;">
+					<div class="container container--xl wrapper--pad" style="margin-bottom:70px;">
+						<div class="quiz-creator-intro-copy" style="margin-right:10px;">
+								<<h2>Quiz Creator</h2>
+								<p>Create a quiz to engage your users. Easily embed the quiz on any website.</p>
+								<p>Our research shows that quizzes help people learn more than presenting information without a quiz. Even more, they increase time on page and site visitors rate them as enjoyable. </p>
+								<a href="/research/online-polls-and-quizzes/" style="margin-right: 10px;">Read the report →</a>
 
-								<a class="btn btn-primary" href="https://mediaengagement.org/wp-login.php?action=register">Register</a>  Or, <a href="https://mediaengagement.org/wp-login.php">login</a>
+								<a class="btn btn-primary" style="margin-right:5px;" href="https://mediaengagement.org/wp-login.php?action=register">Register</a> Or <a class="btn btn-primary" href="https://mediaengagement.org/wp-login.php">Login</a>
 						</div>
 						<div class="text-center">
 							<div class="quiz-example"><script src="https://mediaengagement.test/wp-content/plugins/enp-quiz/public/quiz-take/js/dist/iframe-parent.min.js" type="text/javascript"></script>
```

Comment:
There's a reference to our dev site here. For all of these hardcoded site URLs, can you use `{{ site.url }}` (or whatever the corresponding twig definition is to get the current site URL)